src/WithStylesContext.js

```diff
@@ -0,0 +1,8 @@
+import { createContext } from 'react';
+
+const WithStylesContext = createContext({
+  stylesInterface: null,
+  stylesTheme: null,
+});
```

Comment:
@ljharb Can I call `detectHooks` at this level?

Reply:
@ljharb PTAL

Reply:
yes, this looks great.